commons/com.b2international.commons/src/com/b2international/commons/json/Json.java

```diff
@@ -110,5 +134,20 @@ public static Json assign(Map<String, Object>...sources) {
 			return result;
 		}
 	}
+	
+	@SafeVarargs
+	public static Json merge(Map<String, Object>...sources) {
+		if (CompareUtils.isEmpty(sources)) {
+			return new Json(Map.of());
+		} else {
+			Json result = new Json(sources[0]);
```

Comment:
The single-arg version of `merge` handles `null` values correctly, but the first element of the varargs version can still be `null`. `null` input could also be converted to an empty Map in the constructor.